src/sentry/services/hybrid_cloud/integration/service.py

```diff
@@ -109,6 +109,7 @@ def get_organization_integrations(
         organization_id: Optional[int] = None,
         organization_ids: Optional[List[int]] = None,
         status: Optional[int] = None,
+        statuses: Optional[List[int]] = None,
```

Comment:
Do you have use cases for this outside of tests?  We've been trying to keep the number of filter options limited as each filter option we add complicates caching being added in the future.

Reply:
@markstory thanks for chiming in. That makes sense. I would use it [here](https://github.com/getsentry/sentry/pull/52949/files#r1265773718), but given your point I think it may make sense to just continue to filter down after making the query. There most likely won't be many org integrations returned by my query anyway, so no biggie in terms of efficiency.

That being said, I think there's an argument to be made in favour of `statuses` over `status`, if we want only one of them. 

Let me know your thoughts!